geo/walk_test.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Tests for speak.py'''
+
+from geo import walk
+
+import unittest
+from shapely.geometry.point import Point
+
+class WalkTest(unittest.TestCase):
+
+  def testSingleOutput(self):
+    start_point=(40.748432,-73.982473)
+    end_point=(40.74907,-73.98403)
+    list_points = walk.compute_route(start_point,end_point)
```

Comment:
Add the actual points as a variable (predefined) and then check that they all match. (will work trivially now, but will serve to check that things don't fail later if compute_route changes.)

the bounding box check is still relevant as that may succeed while the individual point check could fail.